resources/ibc-go/default.nix

```diff
@@ -54,13 +54,4 @@ in
         tags = ["netgo"];
         excludedPackages = ["./e2e"];
       };
-
-      ibc-go-v7-simapp = {
```

Comment:
Why the rename to `simd` here?

Reply:
No particular reason, I will revert it to the previous name